backend/src/main/java/com/jobwait/persistence/PostgresController.java

```diff
@@ -0,0 +1,179 @@
+package com.jobwait.persistence;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.UUID;
+
+import com.jobwait.domain.Answers;
+import com.jobwait.domain.User;
+import com.jobwait.domain.ValidEducationLevel;
+import com.jobwait.domain.ValidWorkContract;
+import com.jobwait.domain.ValidWorkModel;
+import com.jobwait.persistence.adapters.PostgresUserAdapter;
+
+public class PostgresController extends PersistenceController {
+    private String jdbcUrl = "jdbc:postgresql://localhost:5432/mydatabase";
+    private String dbUser = "postgres";
+    private String dbPassword = "password";
+
+    private Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(jdbcUrl, dbUser, dbPassword);
+    }
+
+    @Override
+    public User getUserFromAuthId(String authId) {
+        try {
+            Connection connection = getConnection();
+            PreparedStatement statement = connection.prepareStatement("SELECT * FROM users WHERE authhash = ?");
+            statement.setString(1, authId);
+            ResultSet resultSet = statement.executeQuery();
+            List<User> users = Util.resultSetRowsToAdaptedRows(resultSet, new PostgresUserAdapter());
+            return Util.assertSingleElement(users);
+        } catch (ElementNotFoundException e) {
+            throw new RuntimeException(String.format("Could not find user with authId: %s", authId));
```

Comment:
nice message

Reply:
This will eventually be packaged into a response. Currently the backend just shits itself and displays an error, we want to actually send this message back to the client so they know what's up, probably with some kind of error code included (`USER_NOT_FOUND`) so the frontend can interpret it more easily and show some kind of error popup. /ramble